satpy/tests/reader_tests/test_seviri_l1b_native.py

```diff
@@ -1248,7 +1252,144 @@ def test_read_header():
     dtypes = np.dtype([(k, t) for k, t in zip(keys, types)])
     hdr_data = np.array([values], dtype=dtypes)
 
-    with mock.patch("satpy.readers.seviri_l1b_native.np.fromfile") as fromfile:
+    with mock.patch("satpy.readers.seviri_l1b_native.fromfile") as fromfile:
         fromfile.return_value = hdr_data
         actual = recarray2dict(hdr_data)
     assert actual == expected
+
+
+@pytest.fixture(scope="session")
+def session_tmp_path(tmp_path_factory):
+    """Generates a single temp path to use for the entire session."""
+    return tmp_path_factory.mktemp("data")
+
+
+@pytest.fixture(scope="session")
+def tmp_seviri_nat_filename(session_tmp_path):
+    """Creates a fully-qualified filename for a seviri native format file."""
+    full_file_path = session_tmp_path / "MSG4-SEVI-MSG15-0100-NA-20210528075743.722000000Z-N.nat"
+    create_physical_seviri_native_file(full_file_path)
+    return full_file_path
+
+
+@pytest.fixture(scope="session")
+def compress_seviri_native_file(tmp_seviri_nat_filename, session_tmp_path):
+    """Compresses the given seviri native file into a zip file."""
+    zip_full_path = session_tmp_path / "test_seviri_native.zip"
+    with zipfile.ZipFile(zip_full_path, mode="w") as archive:
+        archive.write(tmp_seviri_nat_filename, os.path.basename(tmp_seviri_nat_filename))
+    return f"zip://*.nat::{zip_full_path}"
```

Comment:
Tests are failing on windows because of this.

Reply:
Just pushed a fix. Let's see if it actually works!

Reply:
CI/tests on Windows succeeds!